.gitignore

```diff
@@ -1,6 +1,10 @@
 *.egg-info/
-*.pyc
-/.eggs/
-/.tox/
+*.py[co]
 .coverage
 .coverage.*
+.tox/
+/.eggs/
+/.tox/
```

Comment:
Minor: it's a bit weird to see two different .tox/ exclude rules.

Reply:
Whoops, I assumed it was there. The classifier was, and it was in tox.ini.  Thanks, I'll do that.